src/index.js

```diff
@@ -77,7 +78,7 @@
             }
         };
         el.src = src;
-        doc.documentElement.appendChild(el);
+        head && head.appendChild(el);
```

Comment:
I'm not sure I like silent failing—it would be better if lack of the header results in an error that we can track with something like appdynamics.